### ISSUE TITLE
Some pointers on what to do with the icons

### ISSUE THREAD

**Issue by GijsTimmers** [GijsTimmers/kotnetcli#74]
Sam has recently sent us a preview of the icon set for kotnetcli. Jo and I think that it looks great. We need to make a list of preferred icons, so that Sam has a good idea where his efforts are most needed.

Feel free to edit this list at any time.

We should have:

- [ ] A banner for `README.md`
- [ ] A normal application icon in SVG format. We should be able to resize it to multiple sizes: 16x16, 24x24, 48x48... you get the idea
- [ ] An app icon, preferably with rounded corners
- [ ] A monochrome ('black-white') icon for Unity' and Gnome's top bar, able to show connection status:
  - [ ] Disconnected
  - [ ] Connecting
  - [ ] Connected
- [ ] A Windows traybar icon: low-priority
- [ ] A Mac traybar icon: maybe this can be just the same as the monochrome Unity/Gnome icon. As we don't own a Mac, we don't know if this looks good or not.

For the 'Connecting' icon: it would be nice if the two K's in the monochrome icon could alternatingly switch from black to white indicate that `kotnetcli` is trying to connect. As soon as the connection is succesful, stop switching colour and have the front K in the colour black, and the back K in the colour white. We are not sure on how to indicate the 'Disconnected' status: maybe have two black K's? Or two grey K's? Feel free to put in some thoughts here.

**Comment by jovanbulck** [comment]
thx @samgielis

**Comment by jovanbulck** [comment]
Note on the monochrome icon: we can't use pure black (at least in Gnome) as the background is black; the icon should use white and (a single tint of) gray, as shown in the screenshot below

![screenshot_gnome3](https://cloud.githubusercontent.com/assets/2464627/6530460/657c7ae0-c431-11e4-8b8c-36df35ae9e0e.png)

**Comment by GijsTimmers** [comment]
Good catch!

**Comment by samgielis** [comment]
consider all of this done - in the next week(s) or so :)

**Comment by GijsTimmers** [comment]
You're the best!

**Comment by GijsTimmers** [comment]
@samgielis do you mind if I use the currently available icons for `kotnetcli` Android app development? I received the 1000x1000 icon and banner by email from Jo. Of course, I will mention your name and Github page in the _About_ section.

**Comment by samgielis** [comment]
@GijsTimmers  no, by all means, use them! Sorry I never got to creating those icons. Do you have them covered or could you still use some help?

**Comment by GijsTimmers** [comment]
Thank you. The stuff we have right now will do for a while.

**Comment by jovanbulck** [comment]
I created an ascii version of the iconic double key - for kotnetcli should not forget its roots as a command line application. Inspired by `pacman --version`, I figured we could display the ascii logo on the left when invoking `kotnetcli --version`. Here we go:

```
jo@Giedion[0]::~$ cat ~/Desktop/kotnetcli_ascii_logo.txt 
   __
 __\ \                 kotnetcli v1.3.0-dev
 \ \\_\___             Copyright (C) 2014-2016 Kotnetcli Development Team
  \   V  / __          (https://github.com/GijsTimmers/kotnetcli)
   \  `\<</ /
    \  _\_\<<          This program may be freely redistributed under
     \_\ `_\_\         the terms of the GNU General Public License.
        \_\

jo@Giedion[0]::~$ 
```

Looks better in a fixed-with  white-on-black terminal imo:

![kotnetcli_ascii_terminal](https://cloud.githubusercontent.com/assets/2464627/13034049/e3b33ed6-d32a-11e5-9242-9123983f5e07.png)

I'll open a pull request for this, once the GPL re-licensing #98 is completed.

**Comment by GijsTimmers** [comment]
That's so awesome and so linuxy! thanks!
